tests/crowdin.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { updateOrCreateFile } from '../src';
+import Crowdin, { ResponseObject, SourceFilesModel, UploadStorageModel } from '@crowdin/crowdin-api-client';
+import { createMock } from 'ts-auto-mock';
+describe('UpdateOrCreateFile function', () => {
+    it('should create file when source file is undefined', async () => {
+        let spyProjectId: number;
+        let spyRequest: SourceFilesModel.CreateFileRequest;
+        const client = createMock<Crowdin>({
```

Comment:
I guess we can create this mock on one level above and not per each test

Reply:
ok